README.md

```diff
@@ -11,7 +11,7 @@ lifecycle of Google Compute Engine Persistent Disks.
 
 ## Project Status
 
-Status: Beta
+Status: GA
 Latest stable image: `gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver:v0.7.0-gke.0`
```

Comment:
should it point to 1.0.0?

Reply:
Sure. I guess it'll be broken until the release process completes.